### ISSUE TITLE
Links with the same name should have similar purpose. The link to download each letter is named "Download letter (PDF)" which makes it difficult for screen reader users to determine which letter they are downloading. The link name could include the date. This is not called in out in the Test Rails tests, likely because it is WCAG AAA, but it does not meet WCAG Success Criterion 2.4.9.

### ISSUE THREAD

**Issue by jacobworrell** [department-of-veterans-affairs/vets-website#22634]

**Comment by skylerschain** [comment]
See @joshkimux's last comment on [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/46508#issuecomment-1297661689) for more context about our design decisions here to help resolve these a11y concerns. @jaredcunha

**Comment by skylerschain** [comment]
This ticket should be fixed. The solution can be tracked [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/46508#issuecomment-1297661689) (tl;dr we added dates to links to make them unique).
